pennylane/noise/conditionals.py

```diff
@@ -747,7 +746,7 @@ def _partial(wires=None, **partial_kwargs):
                 op_args[key] = val
 
         if issubclass(op_class, qml.operation.Operation):
-            num_wires = getattr(op_class, "num_wires", AnyWires)
+            num_wires = getattr(op_class, "num_wires", None)
```

Comment:
Unrelated to this PR, but won't all `Operation` subclasses be guaranteed to have a `num_wires` attribute?

Reply:
Yes. But I don't want to go down a rabbit hole trying to understand this module.